docs/UserGuide.md

```diff
@@ -113,18 +113,15 @@ Examples:
 ### Adding a meeting : `addMeeting`
 
 Adds a new meeting to MyInsuRec. 
-DATE should be in DDMMYYYY format and TIME should be in 24-hour format.
+DATE should be in DD-MM-YYYY format and TIME should be in 24-hour format.
 
-Format: `addMeeting i/INDEX d/DATE t/TIME dn/DESCRIPTION`
+Format: `addMeeting n/NAME d/DATE t/TIME dn/DESCRIPTION`
```

Comment:
Just putting here as a reminder that after this iteration, we should use `i/` instead